test-app/tests/integration/components/prismic/element-test.js

```diff
@@ -22,11 +22,13 @@ module('Integration | Component | prismic/element', function (hooks) {
       };
       this.onUnknownTag = (node) => {
```

Comment:
I'm thinking why not installing and using `sinon` to create a `spy`? But for just one instance it might be overkill, right?

Reply:
@vscav yeah, that was my reasoning as well. I also explored [assert.async()](https://api.qunitjs.com/assert/async/), but in the end this seemed the most pragmatic approach.